src/Microsoft.Identity.Web/WebApiExtensions/MicrosoftIdentityWebApiAuthenticationBuilderExtensions.cs

```diff
@@ -178,6 +178,7 @@ public static class MicrosoftIdentityWebApiAuthenticationBuilderExtensions
                     {
                         mergedOptions.Authority = AuthorityHelpers.BuildCiamAuthorityIfNeeded(mergedOptions.Authority, out bool preserveAuthority);
                         mergedOptions.PreserveAuthority = preserveAuthority;
+                        AuthorityHelpers.AddAuthorityQueryToOptions(mergedOptions);
```

Comment:
Need to update MicrosoftIdentityWebAppAuthenticationBuilderExtensions also

Reply:
Done!